ddtrace/tracer/option.go

```diff
@@ -262,8 +269,18 @@ func newConfig(opts ...StartOption) *config {
 		c.transport = newHTTPTransport(c.agentAddr, c.httpClient)
 	}
 	if c.propagator == nil {
+		envKey := "DD_TRACE_X_DATADOG_TAGS_MAX_LENGTH"
```

Comment:
Since this env. key is used in multiple places, should we make it into a constant? Granted we should do that with all of them...

Reply:
This env key isn't actually used anywhere else as far as my searching can see? Am I missing a spot here? 

(The reason I didn't make it a constant to begin with was it only exists in this one spot afaik)

Reply:
You're right! My bad!